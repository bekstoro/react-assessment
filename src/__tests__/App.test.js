import React from 'react';
import { render } from '@testing-library/react';

import App from '../App';

describe('App', () => {
  it('App renders correctly', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
