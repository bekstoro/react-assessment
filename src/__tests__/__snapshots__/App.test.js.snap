// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders correctly 1`] = `
<div>
  <nav
    class="Nav__NavWrapper-sc-13du1es-0 cZYmJv"
  >
    <a
      aria-current="page"
      class="Nav__NavButton-sc-13du1es-1 idRylV active"
      href="/"
    >
      Task One
    </a>
    <a
      class="Nav__NavButton-sc-13du1es-1 idRylV"
      href="/task-two"
    >
      Task Two
    </a>
  </nav>
  <div
    class="Task-sc-7z955n-0 dBJrLg"
  >
    <h1
      class="Task__Heading-sc-7z955n-1 hiEtuP"
    >
      Task One
    </h1>
    <div>
      <div>
        Filter:
        <input
          placeholder="Enter username"
          type="text"
          value=""
        />
      </div>
      <div
        class="UserList__Users-g96ac2-2 fpNOEP"
      />
    </div>
  </div>
</div>
`;
